package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "make lint && make build",
+    "test": "make lint && make test && make build",
```

Comment:
Do we ever use `npm test`?

Reply:
I was using it locally to kick off linting and testing at the command line. Seems like we can either just remove the whole thing or at the very least add the new test to the `npm test` script, yeah? I'm fine with either.